### PR TITLE
Add Twig_ExistsLoaderInterface back for 1.x BC layer #1644

### PR DESCRIPTION
**Pull request by joelpittet**
This is the 2.x change for 1.x BC layer of Twig_ExistsLoaderInterface for #1644

### CODE REVIEW

File: lib/Twig/ExistsLoaderInterface.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+/**
+ * Adds an exists() method for loaders. Extends Twig_LoaderInterface for loaders
+ * can be compatible with Twig 1.x and 2.x.
+ *
+ * 1.x BC layer.
+ *
+ * @author Florin Patan <florinpatan@gmail.com>
+ *
+ * @deprecated since 1.12 (to be removed in 3.0)
```
Comment (stof) [inline lib/Twig/ExistsLoaderInterface.php]: it is not deprecated in 1.12 but in 2.0 (in 1.12, it is the only way to get this feature)

Comment (joelpittet) [inline lib/Twig/ExistsLoaderInterface.php]: Thanks, I thought so but I copied that from what is in 1.x, likely need to change in both branches?

**Comment by stof** [comment]
:+1:

File: lib/Twig/ExistsLoaderInterface.php (hunk 1 above)
Comment (stof) [inline lib/Twig/ExistsLoaderInterface.php]: yeah, the deprecation message in 1.x looks wrong too then.

File: lib/Twig/ExistsLoaderInterface.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ * This file is part of Twig.
+ *
+ * (c) 2009 Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * Adds an exists() method for loaders. Extends Twig_LoaderInterface for loaders
```
Comment (Tobion) [inline lib/Twig/ExistsLoaderInterface.php]: language problem: "for loaders can be"

Comment (stof) [inline lib/Twig/ExistsLoaderInterface.php]: this does not add this method anymore :)

**Comment by fabpot** [comment]
Thank you @joelpittet.

**Pull request merged**